mirascope/core/mistral/dynamic_config.py

```diff
@@ -1,9 +1,8 @@
 """This module defines the function return type for functions as LLM calls."""
 
-from mistralai.models.chat_completion import ChatMessage
-
 from ..base import BaseDynamicConfig, BaseMessageParam
 from .call_params import MistralCallParams
+from .message import ChatMessage
 
 MistralDynamicConfig = BaseDynamicConfig[
     ChatMessage | BaseMessageParam, MistralCallParams
```

Comment:
this would then just be a union of all the message types + `BaseMessageParam` since they don't have an internal union type?

Reply:
Yeah. Do you want to explicitly break it out like:

```
MistralDynamicConfig = BaseDynamicConfig[
    SystemMessage | AssistantMessage | UserMessage | ToolMessage | BaseMessageParam, MistralCallParams
```

or is this just an ask for clarification?

Reply:
yeah breaking it out like you showed is what I meant